### PR TITLE
vendor containerd, BuildKit, protobuf, grpc, and golang.org/x

### PR DESCRIPTION
**Pull request by AkihiroSuda**
containerd/containerd changes: https://github.com/containerd/containerd/compare/acdcf13d5eaf0dfe0eaeabe7194a82535549bc2b...4d242818bf55542e5d7876ca276fea83029e803c
moby/buildkit changes: https://github.com/moby/buildkit/compare/d75ed2b682485d4e4a35e46531230111424f8a65...4d8d91bf49c769b8458e9aa84746c842b4a0e39a

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -86,7 +86,11 @@ github.com/opencontainers/image-spec                d60099175f88c47cd379c4738d15
 github.com/seccomp/libseccomp-golang                689e3c1541a84461afc49c1c87352a6cedf72e9c # v0.9.1
 
 # systemd integration (journald, daemon/listeners, containerd/cgroups)
+github.com/coreos/go-systemd/v22                    2d78030078ef61b3cae27f42ad6d0e46db51b339 # v22.0.0
+github.com/godbus/dbus/v5                           37bf87eef99d69c4f1d3528bd66e3a87dc201472 # v5.0.3
+# go-systemd v17 is required by github.com/coreos/pkg/capnslog/journald_formatter.go
```
Comment (AkihiroSuda) [inline vendor.conf]: Note: dependency on `pkg/caplog` can be removed by updating etcd to master (v3.5) https://github.com/etcd-io/etcd/commit/725e09023ad481c8dc05e2ecce596fb6c7413f5c

File: vendor.conf [hunk 2]
```diff
@@ -12,22 +12,22 @@ github.com/konsorten/go-windows-terminal-sequences  f55edac94c9bbba5d6182a4be46d
...
 
 github.com/RackSec/srslog                           a4725f04ec91af1a91b380da679d6e0c2f061e59
 github.com/imdario/mergo                            1afb36080aec31e0d1528973ebe6721b191b0369 # v0.3.8
-golang.org/x/sync                                   e225da77a7e68af35c70ccbf71af2b83e6acac3c
+golang.org/x/sync                                   cd5d95a43a6e21273425c7ae415d3df9ea832eeb
 
 # buildkit
-github.com/moby/buildkit                            d75ed2b682485d4e4a35e46531230111424f8a65
-github.com/tonistiigi/fsutil                        6c909ab392c173a4264ae1bfcbc0450b9aac0c7d
+github.com/moby/buildkit                            4d8d91bf49c769b8458e9aa84746c842b4a0e39a
```
Comment (thaJeztah) [inline vendor.conf]: Are there fixes and/or user-facing changes in this buildkit update?

Comment (AkihiroSuda) [reply vendor.conf]: Vendor changes and https://github.com/moby/buildkit/pull/1381

**Review by cpuguy83 — CHANGES_REQUESTED** [review]
Needs rebase :( but otherwise LGTM

**Comment by AkihiroSuda** [comment]
rebased

**Review by cpuguy83 — APPROVED** [review]
LGTM

**Comment by AkihiroSuda** [comment]
@tonistiigi @tiborvass PTAL?

**Comment by AkihiroSuda** [comment]
CI seems down

**Comment by AkihiroSuda** [comment]
CI green

@thaJeztah PTAL?

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**